Runs: Rewrite Runs section (#2011)
wandb/docs: models/runs/customize-run-display.mdx

---

> title: View all runs a project

Reviewer: ```suggestion
title: View all runs a project
sidebarTitle: View all runs
```
I think it will improve the left nav to shorten some of these page nav titles.

> description: This section shows how to customize how runs are displayed in your project's runs table

Reviewer: ```suggestion
description: Details about customizing how runs are displayed in your project's runs table
```

---

{/* <Note>
A workspace is limited to displaying a maximum of 1000 runs, regardless of its configuration.
</Note> */}

> View all runs logged to your W&B project in the **Runs** tab of your project sidebar. The *Runs table* shows details about all of your runs in a project. Use the Runs table to compare runs, sort runs by specific columns, and organize runs into groups.

Reviewer: To connect the dots, is it worth mentioning that the Runs Table is in the Runs Tab, and that the Runs Table and Workspace runs list share a config?

The following image shows the Runs table for a project named `deep-drive`:

<Frame>
    <img src="/images/runs/run_table_all.png" alt="Runs table"  />
</Frame>

> ## Add columns

Reviewer: These sections could be consolidated as H3s in a "Manage columns" section, all the way through Hide Columns. This would help the right-hand TOC.

---
title: View runs in a project
description: Details about customizing how runs are displayed in your project's runs table
---

{/* <Note>
A workspace is limited to displaying a maximum of 1000 runs, regardless of its configuration.
</Note> */}

> View all runs logged to your W&B project in the **Runs** tab of your project sidebar. Within the Runs tab is the *Runs table*. The Runs table shows details about all of your runs in a project. Use the Runs table to compare runs, sort runs by specific columns, and organize runs into groups.

Reviewer: What about "Within the Runs tab, the Runs table shows...."

The following image shows the Runs table for a project named `deep-drive`:

<Frame>
    <img src="/images/runs/run_table_all.png" alt="Runs table"  />
</Frame>

## Manage columns

> The following sections describe how to add, remove, move, pin, hide, and sort columns in the Runs table.

Reviewer: What about "how to customize the Runs table"?
